### ISSUE TITLE
Provide ENUM values for operations param in REST component

### ISSUE THREAD

**Issue by kahboom** [KaotoIO/kaoto-backend#635]
**Is your feature request related to a problem? Please describe.**
Yes, I am needing to type out each operation for the REST component.

![REST-dropdown](https://user-images.githubusercontent.com/3844502/235694125-229ab44c-8d2b-429e-ab1b-3ecc360c411e.png)

It seems the ENUM is empty, but should contain the operations there.

**Describe the solution you'd like**
I would like to have the operations like `GET`, `POST`, `PUT`, and `DELETE` available from a dropdown menu. From the frontend perspective, we can use the same data structure to have the dropdown generated automatically in the form.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by lhein** [comment]
I would rephrase this issue to "Provide ENUMs where enums are defined in the catalog"
